js/module.js

```diff
@@ -440,6 +441,48 @@ var Module = Class.extend({
 	}
 });
 
+/** Merging MagicMirror (or other) default/config script
+ * merge 2 objects or/with array
+ * using:
+ * -------
+ * this.config = configMerge({}, this.defaults, this.config)
+ * -------
+ * arg1: initial objet
+ * arg2: config model
+ * arg3: config to merge
+ * -------
+ * why using it ?
+ * Object.assign() function don't to all job
+ * it don't merge all thing in deep
+ * -> object in object and array is not merging
+ * -------
+ * @bugsounet
+ * @Todo: idea of Mich determinate what do you want to merge or not
+ */
+
```

Comment:
could you remove that empty line and adjust the comment to look like jsdoc? so that the documentation is the same everyhwere?